src/server/src/main/java/io/cassandrareaper/resources/RepairRunResource.java

```diff
@@ -343,22 +354,24 @@ public Response modifyRunState(
       return Response.status(Response.Status.BAD_REQUEST.getStatusCode()).entity("\"state\" argument missing").build();
     }
 
-    final Optional<RepairRun> repairRun = context.storage.getRepairRun(repairRunId);
-    if (!repairRun.isPresent()) {
+    final Optional<RepairRun> maybeRepairRun = context.storage.getRepairRun(repairRunId);
```

Comment:
not too sure why you're introducing and renaming `maybeRepairRun`… (or `maybeRepairUnit` below)
these touches feel superfluous to the patch's objective…

Reply:
It's just a DRY implementation.
There were a lot of places with `repairRun.get()` below `!repairRun.isPresent()` check. In my opinion it's clearly to read.
May you explain what's wrong?

Reply:
it's not related to the PR, and the statement on this line isn't otherwise touched in the PR.
if you'd like to offer code style/formatting improvements that should be done as a separate PR.